tests/ext-popup.test.js

```diff
@@ -23,187 +26,205 @@ function observeChange(element) {
   });
 }
 
-test('start monitoring all extensions from the popup', async () => {
+const getMonitorStatusSpyFn = jest.spyOn(ExtListen, 'getMonitorStatus');
+const startMonitorSpyFn = jest.spyOn(ExtListen, 'startMonitor');
+const stopMonitorSpyFn = jest.spyOn(ExtListen, 'stopMonitor');
+
+afterEach(() => {
+  getMonitorStatusSpyFn.mockReset();
+  startMonitorSpyFn.mockReset();
+  stopMonitorSpyFn.mockReset();
+});
+
+test('clicking on start monitoring button should display that the extensions are being monitored', async () => {
   document.body.innerHTML = popupBody;
 
-  const getMonitorStatusFn = jest.spyOn(ExtListen, 'getMonitorStatus');
-  const startMonitorFn = jest.spyOn(ExtListen, 'startMonitor');
+  // no extensions are being monitored initially
```

Comment:
Added comment here as the line - 
```js
getMonitorStatusSpyFn.mockResolvedValueOnce(false);
```
doesn't explains what's happening if it's being resolved with boolean value. 
This improves the readability of this part.